Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/CommandLine.kt

```diff
@@ -25,75 +25,78 @@ const val TEMP_DIR = "Xtemporary-files-dir"
 
 // TODO: unify camel and snake cases.
 // Possible solution is to accept both cases
-fun getCommonInteropArguments() = listOf(
-        OptionDescriptor(ArgType.Boolean(), "verbose", description = "Enable verbose logging output", defaultValue = "false"),
-        OptionDescriptor(ArgType.Choice(listOf("jvm", "native", "wasm")),
-                "flavor", description = "Interop target", defaultValue = "jvm"),
-        OptionDescriptor(ArgType.String(), "pkg", description = "place generated bindings to the package"),
-        OptionDescriptor(ArgType.String(), "output", "o", "specifies the resulting library file", defaultValue = "nativelib"),
-        OptionDescriptor(ArgType.String(), "libraryPath", description = "add a library search path",
-                isMultiple = true, delimiter = ","),
-        OptionDescriptor(ArgType.String(), "staticLibrary", description = "embed static library to the result",
-                isMultiple = true, delimiter = ","),
-        OptionDescriptor(ArgType.String(), "generated", description = "place generated bindings to the directory",
-                defaultValue = System.getProperty("user.dir")),
-        OptionDescriptor(ArgType.String(), "natives", description = "where to put the built native files",
-                defaultValue = System.getProperty("user.dir")),
-        OptionDescriptor(ArgType.String(), "library", "l", "library to use for building", isMultiple = true),
-        OptionDescriptor(ArgType.String(), "repo", "r",
-                "repository to resolve dependencies", isMultiple = true),
-        OptionDescriptor(ArgType.Boolean(), NODEFAULTLIBS, description = "don't link the libraries from dist/klib automatically",
-                defaultValue = "false"),
-        OptionDescriptor(ArgType.Boolean(), PURGE_USER_LIBS, description = "don't link unused libraries even explicitly specified",
-                defaultValue = "false"),
-        OptionDescriptor(ArgType.String(), TEMP_DIR, description = "save temporary files to the given directory")
-    )
+open class CommonInteropArguments(val argParser: ArgParser) {
+    val verbose by argParser.option(ArgType.Boolean, description = "Enable verbose logging output", defaultValue = false)
+    val flavor by argParser.option(ArgType.Choice(listOf("jvm", "native", "wasm")), description = "Interop target",
+            defaultValue = "jvm")
```

Comment:
BTW, this shall be changed to `native`.

Reply:
As I understood here is jvm, because this flavor is default in `NativeInteropPlugin` which we actively use in compiler exactly for jvm.